src/services/datahub-client.doubles.ts

```diff
@@ -44,4 +44,11 @@ export class HardcodedDatahubClient implements DatahubClient {
     }
     return domain.pages;
   }
+
+  public async getVisitsByPeriod(domainName: number, dateFrom: Date) {
+    console.log('getVisitsByPeriod', domainName, dateFrom);
```

Comment:
If you use an object, we will get better debug information:

```suggestion
    console.log('getVisitsByPeriod', { domainName, dateFrom });
```

is the same than:

```
    console.log('getVisitsByPeriod', { domainName: domainName, dateFrom: dateFrom });
```

Reply:
changed it